osconfig_tests/test_suites/package_management/package_management_test_data.go

```diff
@@ -40,19 +40,27 @@ func addCreateOsConfigTest(pkgTestSetup []*packageManagementTestSetup) []*packag
 	desc := "test osconfig creation"
 	for _, pkgManager := range pkgManagers {
 		var oc *osconfigpb.OsConfig
+		var image string
 
 		switch pkgManager {
 		case "apt":
+			image = debianImage
 			pkg := osconfigserver.BuildPackage("cowsay")
 			pkgs := []*osconfigpb.Package{pkg}
-			oc = osconfigserver.BuildOsConfig(testName, desc, osconfigserver.BuildAptPackageConfig(pkgs, nil, nil), nil, nil, nil, nil)
+			oc = osconfigserver.BuildOsConfig(fmt.Sprintf("%s-%s", filepath.Base(image), testName), desc, osconfigserver.BuildAptPackageConfig(pkgs, nil, nil), nil, nil, nil, nil)
+			break
+		case "yum":
+			image = centosImage
+			pkg := osconfigserver.BuildPackage("cowsay")
```

Comment:
It might be helpful to make the package name a constant - possibly one for each distro in case we need to change the package name. It's also self documenting code for people not familiar with "cowsay".

Reply:
yeah, i agree, this hardcoding will be removed when i merge my other PR https://github.com/GoogleCloudPlatform/compute-image-tools/pull/677